Js-Games/Frog Hop/README.md

```diff
@@ -0,0 +1,8 @@
+## Frog Hop
+
+It is a multiplayer javascript game in which one user plays as green frogs while the other one plays as yellow frogs. The task is to eliminate the frogs of other player and protect your own frogs from attacks.
+
+## Setting Up
+- Download the images/gif from here: https://drive.google.com/drive/folders/12yStxytssDNZwsVZttXQ2EFp50zYDNaT?usp=sharing
```

Comment:
You not need ti do this, you have to generate link of individual image and then instead of image path use that link in src to load image in project

Reply:
@MohitGupta121 done!